contrib/storage-mongo/src/main/java/org/apache/drill/exec/store/mongo/MongoGroupScan.java

```diff
@@ -180,39 +179,39 @@ private void init() throws IOException {
     chunksMapping = Maps.newHashMap();
     chunksInverseMapping = Maps.newLinkedHashMap();
     if (isShardedCluster(client)) {
-      MongoDatabase db = client.getDatabase(CONFIG);
-      MongoCollection<Document> chunksCollection = db.getCollection(CHUNKS);
+      MongoDatabase db = client.getDatabase(DrillMongoConstants.CONFIG);
```

Comment:
Can you use a static import here so we don't have a bunch of extra noise in the patch?